lithium/lithium.py

```diff
@@ -268,7 +268,7 @@ def readTestcaseLine(line):
 def writeTestcase(filename):
     with open(filename, "w") as file:
         file.write(before)
-        for i in range(len(parts)):
+        for i in xrange(len(parts)):
```

Comment:
Note that range() in python3 is xrange() in python2.7 and 2to3 will [translate xrange to range properly](http://stackoverflow.com/a/135114).

The linter points out [error 0200](https://github.com/codacy/codacy-pylint/blob/master/src/main/resources/docs/description/C0200.md) if we don't fix this.

What alternative do you suggest? Or should we just leave this?